### DISCUSSION TITLE
[feature request] add updates and feedback activity on people I follow, enhance the way the app viewing repos recommendations.

### DISCUSSION THREAD

**Discussion by nadir500** [community/community#3549] [Mobile] [answered]
Is it possible to have the same feedback on activity for the ones I follow on the app? Like what they contributed to and what did they star/watch/followed.

Also it's annoying the way you have to right slide to see recommended repos to explore, maybe put the rest in list view after sliding three cards? 

Another thing is that the recommendation system for showing repos that interest me needs to be improved depending on people I follow and the types of projects I am mainly interested in with trending and for you sections.

**Comment by nadir500** [comment]
> Is it possible to have the same feedback on activity for the ones I follow on the app? Like what they contributed to and what did they star/watch/followed.
> Also it's annoying the way you have to right slide to see recommended repos to explore, maybe put the rest in list view after sliding three cards?
> Another thing is that the recommendation system for showing repos that interest me needs to be improved depending on people I follow and the types of projects I am mainly interested in with trending and for you sections.

Also can we have refreshed new repos to explore every time I refresh explore tab? Because the exact same projects viewed and not being updated until the next day or so.

**Reply by tylerlittlefield** [reply]
100% agree. This is the only reason I don’t use mobile. I need to see activity for those I follow in order to stay in the loop.

**Comment by candyho** [comment, marked as answer]
Hey thanks for the feedback! We hear you. Improving the Explorer tab in the mobile app is something on our radar. Stay tuned!
